src/main/resources/application.properties

```diff
@@ -17,4 +17,7 @@ spring.jpa.open-in-view=false
 
 # Kafka
 kafka.bootstrap-server=${BOOTSTRAP_SERVER}
-kafka.topic=callisto-timecard
\ No newline at end of file
+kafka.topic=callisto-timecard
+
+# Devtools
```

Comment:
Didn't we find this was unnecessary in the end as it was the recompile that made the changes get picked up, not this setting

Reply:
Removed now